packages/meditrak-server/src/apiV2/userAccounts/CreateUserAccounts.js

```diff
@@ -76,7 +78,7 @@ export class CreateUserAccounts extends CreateHandler {
         });
       }
 
-      return { userId: user.id, secretKey };
+      return { ...user, userId: user.id, secretKey };
```

Comment:
Bit of a danger here that we'll expose the hashed password to the front end, because it's a generic helper that might be used in a bunch of places. Let's just pass back what we need, and if we need the hashed password from the user in one of the places that calls this, refetch the user from the db at that point.